saleae/saleae.py

```diff
@@ -96,7 +96,7 @@ class ImpossibleSettings(SaleaeError):
 		pass
 
 	@staticmethod
-	def launch_logic(timeout=15, quiet=False, logic_path=None, args=None):
+	def launch_logic(timeout=15, quiet=False, host=None, port=None, logic_path=None, args=None):
```

Comment:
```suggestion
	def launch_logic(timeout=15, quiet=False, host=localhost, port=10429, logic_path=None, args=None):
```

I think it probably makes sense to just include the defaults again here, then you don't have to worry about any possible checks later

Reply:
ACK.